src/misc/lv_style.c

```diff
@@ -89,17 +89,17 @@ bool lv_style_remove_prop(lv_style_t * style, lv_style_prop_t prop)
     for(i = 0; i < style->prop_cnt; i++) {
         if(old_props[i] == prop) {
             lv_style_value_t * old_values = (lv_style_value_t *)style->v_p.values_and_props;
-            style->prop_cnt--;
 
             uint16_t * new_props = &style->prop1;
             lv_style_value_t * new_values = &style->v_p.value1;
 
-            if(style->prop_cnt > 1) {
-                size_t size = style->prop_cnt * (sizeof(lv_style_value_t) + sizeof(uint16_t));
+            if(style->prop_cnt > 2) {
+                size_t size = (style->prop_cnt - 1) * (sizeof(lv_style_value_t) + sizeof(uint16_t));
                 uint8_t * new_values_and_props = lv_mem_alloc(size);
                 if(new_values_and_props == NULL) return false;
+                style->v_p.values_and_props = new_values_and_props;
 
-                tmp = new_values_and_props + style->prop_cnt * sizeof(lv_style_value_t);
+                tmp = new_values_and_props + (--style->prop_cnt) * sizeof(lv_style_value_t);
```

Comment:
It's subjective but I prefer adding `style->prop_cnt --` in a new line for easier debugging and readability. 
What do you think?

Reply:
Done.